app/services/SubmittedProcessService.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import connectors.SubmittedProcessConnector
+import javax.inject.{Inject, Singleton}
+import models.{RequestOutcome, SaveSubmittedProcessResponse}
+import play.api.libs.json.JsValue
+import uk.gov.hmrc.http.HeaderCarrier
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class SubmittedProcessService @Inject()(connector: SubmittedProcessConnector) {
```

Comment:
Approval in naming

Reply:
This will be a general service that applies to the SubmittedProcesses collection

Reply:
Forgot to mention the GuidanceService which is a general submission service currently has the submitScratchProcess() api

Reply:
As with the connector, there is only one service in the view project so I would use GuidanceService.

Reply:
I don't like putting everything in one service that will grow over time - Scratch, Submitted and Published processes are separate so think we should keep some separation in the code - if we keep services based on what they are working on, it keeps them cleaner and less code in each class, otherwise we just end up with one big monolithic service.  And means two people can work more easily on same code base if a separation